integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -781,9 +781,8 @@ object CommonLoadUtils {
         internalRowOriginal
       }
       for (index <- timeStampIndex) {
-        if (internalRow.getLong(index) == 0) {
-          internalRow.setNullAt(index)
-        } else {
+        // if value zero case spark will fill the timestamp. no need to fill null
```

Comment:
Should Check, internalRow.isNullAt(index) , for actual null or empty data. then insert null for that row

Reply:
ok